### PR TITLE
Draft post-event pages for website

### PR DESCRIPTION
**Pull request by stevenjmesser**
I've done a few things here, which should be represented in the commits:

- retitled the Design System Day 2022 page
- archived the session and speaker information pages in the `/design-system-day/...`directory
- created a `/design-system-day-2023/` directory and index page, to host information about the event
- moved the old session and speaker information pages to the new `/design-system-day-2023/...` directory
- edited the old session and speaker information pages a tad

I still might play with using summary cards to make the session and speaker information clearer on one page (rather than two).

Open to any and all feedback on this, happy to explain or revisit any choices if it helps explain my thinking.

### CODE REVIEW

File: src/community/design-system-day/session-information.md [hunk 1]
```diff
@@ -1,268 +1,7 @@
...
-
-As we embark on this immersive session, prepare to gain actionable insights that empower you to harness the untapped potential of design systems. Together, let's unlock a united future where the synergy of creativity and clarity gives rise to impactful experiences that resonate and endure.
-
-### Fostering Collaboration in design systems
-
-James Carleton
-3pm to 4:30pm
-
-Design systems work best when they are collaborative and flexible instead of a strict set of fixed rules. I would like to share my experience in building a community that encourages collaboration and contribution to our DS.
+To find out about Design System Day, see the [Design System Day](/community/design-system-day/) page.
```
Comment (colinrotherham) [inline src/community/design-system-day/session-information.md:7]: Maybe only say "Design System Day" once?

Just need a new line after it too for our EditorConfig settings (there's a Prettier warning)

```suggestion
See the [Design System Day](/community/design-system-day/) page to find out more.

```

File: src/community/design-system-day/speaker-information.md [hunk 2]
```diff
@@ -1,213 +1,7 @@
...
-Valentine is a Canadian designer specialising in interaction design, and application of design to solving complex problems. Over the years he has dedicated his time to projects and people that attempt to make the world a better place.
-
-### Hassan Nawaz (FanDuel)
-
-Hassan is curious about people, technology, and behaviour, currently working in senior design leadership at FanDuel, MIT. Passionate about design thinking and humanising technology, Hassan has a track record of creating cohesive, data-driven experiences that drive business results.
-
-### James Carleton (PayPal)
-
-James is Senior UX Designer on the Design System (DS) team at PayPal where he’s built and evolved the DS, its team, and community of internal users. James also enjoys contributing to improved accessibility of PayPal products and exploring design tools. James loves following design system industry events and learning from other systems designers.
+To find out about Design System Day, see the [Design System Day](/community/design-system-day/) page.
```
Comment (colinrotherham) [inline src/community/design-system-day/speaker-information.md:7]:
```suggestion
See the [Design System Day](/community/design-system-day/) page to find out more.

```

**Review by colinrotherham — APPROVED** [review]
Looks good to me, assuming it's mostly copy and pasted

Couple of minor suggestions from me and Prettier will want a blank line at the end of documents too, but will approve anyway. Change them when you're ready 🙌

File: src/community/design-system-day/session-information.md (hunk 1 above)
Comment (stevenjmesser) [reply src/community/design-system-day/session-information.md:7]: Ah, you star!

**Pull request merged**